_posts/2020-11-18-homebrew-tap-with-bottles-uploaded-to-github-releases.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Homebrew tap with bottles uploaded to GitHub Releases
+author: dawidd6
+---
+
+## Intro
+
+Since Homebrew [2.5.2](https://github.com/Homebrew/brew/releases/tag/2.5.2) release, it became possible to upload built bottles (binary packages) to GitHub Releases, in addition to standard way - Bintray. The [PR](https://github.com/Homebrew/brew/pull/8410) made to `Homebrew/brew` repository adding this feature was created by me on 20 August 2020 and merged on 15 September. There was a companion [PR](https://github.com/Homebrew/homebrew-test-bot/pull/486) made to `Homebrew/homebrew-test-bot` adding support for setting the base download URL of bottles pointing to specific release on GitHub. It took some time and testing to get a nicely working solution and even more time to get myself to write this post.
```

Comment:
```suggestion
Since the [Homebrew 2.5.2 release](https://github.com/Homebrew/brew/releases/tag/2.5.2), you can upload bottles (binary packages) to GitHub Releases, in addition to the previous standard - Bintray. The [PR](https://github.com/Homebrew/brew/pull/8410) to `Homebrew/brew` was merged on 15 September. A [companion PR](https://github.com/Homebrew/homebrew-test-bot/pull/486) to `Homebrew/homebrew-test-bot` added support for setting the base download URL of bottles pointing to specific release on GitHub.
```